profile-widget/diveeventitem.cpp

```diff
@@ -152,17 +152,21 @@ void DiveEventItem::setupToolTipString()
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
-			name += QString(" (cyl %1)").arg(internalEvent->gas.index+1);
+			name += tr(" (cyl.") + QString(" %1)").arg(internalEvent->gas.index+1);
 	} else if (value) {
 		if (type == SAMPLE_EVENT_PO2 && name == "SP change") {
-			name += QString(":%1").arg((double)value / 1000);
+			name += QString(": %1bar").arg((double)value / 1000, 0, 'f', 1);
+		} else if (type == SAMPLE_EVENT_CEILING && name == "planned waypoint above ceiling") {
+			const char *depth_unit;
+			double depth_value = get_depth_units(value*1000, NULL, &depth_unit);
+			name += QString(": %1%2").arg((int) round(depth_value)).arg(depth_unit);
```

Comment:
having this as in a separate commit would be nice - the others change translation, this adds a feature